server/src/schemas/usersSchema.ts

```diff
@@ -0,0 +1,73 @@
+import { gql } from "apollo-server-express";
+
+export const UsersTypeDefs = gql`
+  type User {
+    id: ID!
+    first_name: String!
+    last_name: String!
+    email: String!
+    type: Type!
+    privilege: Privilege!
+    registration_date: Date!
+    balance: Int!
+    hold: Boolean!
+    holds: [Hold]
+    training_modules: [TrainingModule]
+    year: Int
+    college: String
+    major: String
+    description: String
```

Comment:
What sort of stuff would go in `description`?

Reply:
This was for the little blurb in the about me for labbies/admins.

Reply:
Gotcha. This is more a matter of preference but IMO `aboutMe` might be more self explanatory. Up to you.